lib/log.fz

```diff
@@ -26,4 +26,4 @@
 # In contrast to `say`, `log` returns its argument
 # which makes `log` compose better in same cases.
 #
-public log(s T) => say s; s
+public log(s T) => {say s; s}
```

Comment:
Why is this semicolon ambiguous?

Reply:
Because Fridtjof said so. The second s could be interpreted as part of the feature declaration `public log(s T) => {say s; s}` or something that is executed afterwards `public log(s T) => {say s}; s`

Reply:
Hmm, okay. But when are semicolons not ambiguous then? Only when adding braces?

Reply:
For feature declarations in one line this is correct.

Reply:
Ah, okay. I understand now.